### ISSUE TITLE
Rendering Markdown With a Formatted and Syntax Highlighted View

### ISSUE THREAD

**Issue by viegasfh** [jrnl-org/jrnl#1335]
### Affected Page(s)

  - https://jrnl.sh/en/stable/recipes/

### What Could Be Better?

Many of Jrnl users render their output as Markdown, and when using the option --format md, we don't get a formatted and syntax highlighted view of output. I am proposing this as a feature, but at the moment there is a workaround using an external tool through piping. There are a few of these tools that can render Markdown in a formatted and syntax highlighted view, but the one  I am sharing here is called **mdless**.

This is how I we can use it:

```
jrnl myjournal -1 --format md | mdless -w 70
```
I use a column width of 70, but you can always use the default or adjust as you wish.

With this recipe one can visualize the Markdown output formatted in the command line.

So that you can see the difference,  I am sharing the default output by jrnl here:

![image](https://user-images.githubusercontent.com/290857/132238330-8a25e9fb-d87a-4cab-ac7c-79b90966ef2b.png)

And the formatted output with mdless here:

![image](https://user-images.githubusercontent.com/290857/132238381-ba102566-2fe3-4570-9430-01d13c801ba6.png)

### Other Information

For Windows users I guess they would have to use PowerShell.

I use a lot of Markdown markup in my entries and always render it as Markdown, and because I found this combination useful, I decided to share it and propose it to be included in the documentation.

**Comment by micahellison** [comment]
Thanks for submitting this. I think this would be welcome in the recipes docs. Would you like to make a pull request for this?

**Comment by viegasfh** [comment]
> Thanks for submitting this. I think this would be welcome in the recipes docs. Would you like to make a pull request for this?

Certainly! I will add it to the recipes doc and then I'll make a pull request and wait for suggestions for improvement in case you find it needs to.

**Comment by viegasfh** [comment]
@micahellison , I have already added this to the documentation, but then got a little busy and forgot to ask you about the inclusion of images. Do you have any guidelines with regards to using images (screenshots in this case). I have already used code blocks where I could, but the rendered Markdown I can't show the example without an image screenshot.

What do you suggest? This way I finish and make a pull request.

**Comment by micahellison** [comment]
> @micahellison , I have already added this to the documentation, but then got a little busy and forgot to ask you about the inclusion of images. Do you have any guidelines with regards to using images (screenshots in this case). I have already used code blocks where I could, but the rendered Markdown I can't show the example without an image screenshot.
> 
> What do you suggest? This way I finish and make a pull request.

Hi @viegasfh, I think keeping it short without an image might be best. We don't have visual examples for other examples involving piping, like the grep example at the top of the page. However, linking to the mdless project would definitely be nice -- people can see an example of how it works relatively easily from its readme.

**Comment by viegasfh** [comment]
Hi, @micahellison ! Thanks for the feedback. I had done that already, but thought that perhaps add a screenshot would make it clear. I will re-edit and then make the pull request.

Thanks!
